apps/web/screens/Category/Category.tsx

```diff
@@ -167,6 +170,7 @@ const Category: Screen<CategoryProps> = ({
               >
                 {sortedGroups.map((groupSlug, index) => {
                   const { title, description, articles } = groups[groupSlug]
+                  console.log(title, description)
```

Comment:
console.log!

Reply:
More importantly, it crashes on rendering the front page:

```GraphQL error: Cannot destructure property `fields` of 'undefined' or 'null'.```

Reply:
That is not related to this